pkg/config/config_parser.go

```diff
@@ -36,6 +40,19 @@ func (p *Parser) LoadConfigFile(path string) (*Config, hcl.Diagnostics) {
 	return p.decodeConfig(body, diags)
 }
 
+func ParseProviderSource(requestedProvider *RequiredProvider) (org string, provider string, err error) {
```

Comment:
We could clean this further after https://github.com/cloudquery/cloudquery/pull/909 is merged.
Also see internal issue https://github.com/cloudquery/cloudquery-issues/issues/406